great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -1,17 +1,14 @@
 """TODO: Add docstring"""
```

Comment:
Mind adding the docstring? Now that the usage pattern is slightly different would be good to document how it is intended to be used (using the return value of migrate to access any failed to migrate validations)

Reply:
done!